_projects/2018/sunpy/sunkit-image.md

```diff
@@ -30,38 +29,40 @@ functionality specific to the analysis of solar physics data.
 
 There have been various proposals for adding image processing and manipulation
 code to the SunPy library. SunPy has decided that this functionality will
-instead reside in an affiliated package, tentatively named 'sunkit-image'. This
+instead reside in an affiliated package, named 'sunkit-image'. This
 project will setup this package and implement the initial functionality.
 
 There is various functionality that should be added to 'sunkit-image' some of it
-already developed, some of it not! This project should achieve some or all of
+already developed, some of it not. This project should achieve some or all of
 the following goals (roughly in this order):
 
-1. Port the Multi-Scale Gaussian Normalisation code from [#1899](https://github.com/sunpy/sunpy/pull/1899).
-2. Convert the [differential rotation code](https://github.com/sunpy/sunpy/blob/master/sunpy/physics/differential_rotation.py) in SunPy to use [`sunpy.coordinates`](https://github.com/sunpy/sunpy/tree/master/sunpy/map).
-3. Implement image warping for solar differential rotation. [#1876](https://github.com/sunpy/sunpy/pull/1876).
-4. Implement the [OCCULT-2 algorithm](http://arxiv.org/abs/1307.5046) for coronal loop tracing.
-5. Implement running and base difference functionality and the persistence transform. See Figure 2 in [this paper](http://iopscience.iop.org/article/10.1088/0004-637X/736/2/102/pdf) for some ideas.
+1. Implement the [normalizing-radial-graded filter (NRGF)](http://adsabs.harvard.edu/abs/2006SoPh..236..263M).
+2. Port the Multi-Scale Gaussian Normalisation (MGN) code from [#1899](https://github.com/sunpy/sunpy/pull/1899).
+3. Implement the [OCCULT-2 algorithm](http://arxiv.org/abs/1307.5046) for coronal loop tracing.
+4. Implement image re-sampling as described [here](https://link.springer.com/content/pdf/10.1023/B:SOLA.0000021743.24248.b0.pdf).
 
 optional extras:
 
+5. Implement the [soft morphological filtering of solar images](https://www.aanda.org/articles/aa/pdf/2006/38/aa4852-06.pdf)
 6. Refactor and write a Python wrapper for [FLCT](https://arxiv.org/abs/0712.4289) [code](http://solarmuri.ssl.berkeley.edu/overview/publicdownloads/software.html).
-8. Implement image alignment using feature detection and tracking. [Example](http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_brief.html)
+7. Implement image alignment using feature detection and tracking. [Example](http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_brief.html)
 
 
 #### Milestones
 
-##### GSOC 2017 CODING STARTS
+##### GSOC 2018 CODING STARTS
```

Comment:
These milestones should probably be updated for the new GSOC evaluation pattern. i.e. start, first, second, final